index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Tarikwa Portfolio</title>
+</head>
+
+<body>
+    <section class="main-section">
+        <section class="toolbar">
+            <ul class="nav">
+                <li class="item"><a href="#">My Logo</a></li>
+                <li class="item"><a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a></li>
+            </ul>
+        </section>
```

Comment:
Nice work implementing the skeletal framework for this project. However, I would like to suggest some areas of improvement:

- [ ] Please change the `section` tag with a class of `main-section` to the `header` tag. This would make the code more semantic. 
- [ ] Kindly wrap the **_hamburger_** icon in a `nav` tag for semantic reasons. You can change the `ul` with class `nav` to a `nav` tag and change the `li` tags to `divs`.

Reply:
I have changed the section to the header as you suggested to me.